test/JDBC/expected/babel_time-vu-verify.out

```diff
@@ -0,0 +1,638 @@
+-- psql
+-- behaviour of casting to time in psql
+SELECT CAST('2012-02-23' AS TIME)
+GO
+~~ERROR (Code: 0)~~
+
+~~ERROR (Message: ERROR: Conversion failed when converting date and/or time from character string.
+  Position: 53
+    Server SQLState: 22007)~~
+
+
+-- tsql
+-- when month is given in nymeric format either date has ['-', '/']
+SELECT CAST('2012-02-23' AS TIME)
+GO
+~~START~~
+time
+00:00:00.0000000
+~~END~~
+
+
+SELECT CAST('2012   -    02   -    23' AS TIME)
```

Comment:
what about spaces in the beginning or end?

Reply:
They will also be truncated, the previous logic works fine for spaces before and after.

Reply:
let us add test cases for the same if not already

Reply:
Added those cases